docs/conf/prague/2020/sponsors/prospectus.rst

```diff
@@ -160,4 +160,116 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
 Run of Show
 ===========
 
-We are still working out the run of show for the virtual event.
+This Run of Show provides more context about the event and answers some common questions you may have.
+Please let us know if there is any information missing that would be useful for you.
+
+Sponsorship schedule
+--------------------
+
+* **SUNDAY**: The conference online platform is open. You are welcome to hang out at your sponsorship booth and attend the Writing Day, but no formal sponsorship events are happening. You're also welcome to lead a Writing Day project if your documentation is open source.
+
+* **MONDAY**: The conference platform opens at 9am, so we recommend arriving around this time to get the most interaction with attendees. This is the official start of the main conference talks, when sponsor booths will need to be staffed. The conference will run until around 6pm.
+
+* **TUESDAY**: The Job Fair will be on Tuesday morning in the Sessions area of the online platform. The normal conference sponsorship activities and expo booths will be available all day Tuesday, but primarily on Tuesday afternoon.
```

Comment:
```suggestion
* **TUESDAY**: The Job Fair will be on Tuesday morning in the Expo area of the online platform. It will take place in existing sponsorship booths. If you do not have a booth, a temporary booth will be set up for the job fair. The normal conference sponsorship activities and expo booths will be available all day Tuesday, but primarily on Tuesday afternoon.
```